tests/providers/apache/hdfs/hooks/test_hdfs.py

```diff
@@ -82,6 +82,8 @@ def test_get_autoconfig_client_no_conn(self, mock_client):
 
     @mock.patch('airflow.providers.apache.hdfs.hooks.hdfs.HDFSHook.get_connections')
     def test_get_ha_client(self, mock_get_connections):
+        if not snakebite_loaded:
+            raise pytest.skip("Skipping test because HDFSHook is not installed")
```

Comment:
We don't need this, or `snakebite_loaded` any longer, right?

Reply:
We don't. But I noticed it too late :)